client/src/components/Ticketing/ticketingmanager/TicketTypes/SubscriptionTypeUtils.tsx

```diff
@@ -0,0 +1,84 @@
+import {useEffect, useState} from 'react';
+import {getData} from '../Event/components/ShowingUtils';
+
+export const validateSubscriptionType = (subscriptionType: any) => {
+  if (+subscriptionType.price <= 0) {
+    throw new Error('Invalid price');
+  }
+  if (subscriptionType.description.length === '') {
+    throw new Error('Invalid Description');
+  }
+  if (subscriptionType.name.length === '') {
+    throw new Error('Invalid Name');
+  }
+  return {
+    description: subscriptionType.description,
+    price: +subscriptionType.price,
+    previewonly: Boolean(subscriptionType.previewonly),
+    name: subscriptionType.name,
+  };
+};
+
+export const getDefaultSubscriptionType = () => ({
+  id: -1,
+  description: '',
+  name: '',
+  price: 0,
+  previewonly: false,
+});
```

Comment:
I think this could just be a constant object rather than a function call.
```
export const defaultSubscriptionType = {
  id: -1,
  description: '',
  name: '',
  price: 0,
  previewonly: false,
};
```

Reply:
Fixed